tests/pytests/pkg/integration/test_version.py

```diff
@@ -84,11 +84,10 @@ def test_salt_versions_report_minion(salt_cli, salt_call_cli, salt_minion):
 @pytest.mark.parametrize(
     "binary", ["master", "cloud", "syndic", "minion", "call", "api"]
 )
-def test_compare_versions(binary, install_salt):
+def test_compare_versions(binary, version, install_salt):
```

Comment:
Wondering why have version passed in, since install_salt has several versions available in it.
Also just grepped and test_compare_versions is not called by anything, so wondering about version getting passed in and used on line 192-191